.circleci/soltest_all.sh

```diff
@@ -30,7 +30,7 @@ REPODIR="$(realpath $(dirname $0)/..)"
 
 for OPTIMIZE in 0 1; do
     for EVM in homestead byzantium constantinople petersburg istanbul; do
-        EVM=$EVM OPTIMIZE=$OPTIMIZE ${REPODIR}/.circleci/soltest.sh
+        EVM=$EVM OPTIMIZE=$OPTIMIZE BOOST_TEST_ARGS="-t !@nooptions" ${REPODIR}/.circleci/soltest.sh
```

Comment:
Maybe we can just add ``--no-smt`` here as well until we have all of them flagged ``nooptions`` (or instead of doing that)?

Reply:
I would prefer to flag it properly. If that is not possible, we can add the switch here.

Reply:
I'd guess the manual test case adding shouldn't make it too hard to do - it'll probably end up a boolean flag in the test suite list or sth.